### PR TITLE
Accordion web widget - add advanced mode features

### PR DESCRIPTION
**Pull request by IIsaku**

### CODE REVIEW

**Comment by IIsaku** [comment]
Requires a technical review before merge.

File: content/appstore/widgets/accordion.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ The [Accordion](https://marketplace.mendix.com/link/component/117895) widget ena
 
 * Displays groups consisting of a header and content
 * Allows you to collapse and expand the content when interacting with the header using a mouse or keyboard
+* Allows you to control the collapsed state of a group through expressions and an entity attribute.
```
Comment (ConnorLand) [inline content/appstore/widgets/accordion.md:17]: Just checking. Do you have to use expressions and an entity attribute **together**, or will either option work?

File: content/appstore/widgets/accordion.md [hunk 2]
```diff
@@ -22,16 +23,28 @@ Configuring the accordion widget is simple. Drag and drop the widget onto a page
...
-  * **Custom header content** (configurable when the header content consists of widgets) – The widgets to be displayed in the header.
-  * **Visible** (true by default) – A Boolean expression indicating whether the group should be visible.
-  * **Dynamic class** (required) – A string expression to apply dynamic classes (separated by a space) to the group.
+* **Groups** (required) – The groups that are part of the accordion. See the following subparagraphs for the configurable properties.
 * **Collapsible** (enabled by default) – Determines whether the content of groups is collapsible.
-* **Expanded groups** (Single by default) – Determines whether one or more groups can be expanded at once.
+* **Expanded groups** ('Single' by default) – Determines whether one or more groups can be expanded at once.
 * **Animate** (enabled by default) – Determines whether the collapsing and expanding of content is animated.
 
+#### 2.1.1 Groups - General Tab
```
Comment (ConnorLand) [inline content/appstore/widgets/accordion.md:31]: I'm going to check if punctuation in headers is okay.

File: content/appstore/widgets/accordion.md (hunk 1 above)
Comment (IIsaku) [reply content/appstore/widgets/accordion.md:17]: Either option will work, so it should be an "or".

File: content/appstore/widgets/accordion.md [hunk 3]
```diff
@@ -22,16 +23,28 @@ Configuring the accordion widget is simple. Drag and drop the widget onto a page
...
+* **Header** ('Text' by default) – Determines if the header content consists of text or widgets.
+* **Text** (configurable when the header content consists of text) – The text value to be displayed in the header.
+* **Render mode** (configurable when the header content consists of text, 'Heading 3' by default) – The text's heading level.
+* **Custom header content** (configurable when the header content consists of widgets) – The widgets to be displayed in the header.
+* **Visible** ('true' by default) – A Boolean expression indicating whether the group should be visible.
+* **Dynamic class** (required) – A string expression to apply dynamic classes (separated by a space) to the group.
+
+#### 2.1.2 Groups > State Tab
+
+* **Start as** ('Expanded' by default) – Determines whether the content of the header starts expanded or collapsed. Optionally through the help of an expression with the 'Dynamic' option.
```
Comment (ConnorLand) [inline content/appstore/widgets/accordion.md:42]: What does this fragment mean? "Optionally through the help of an expression with the 'Dynamic' option."

Does this mean "Can also be used in an expression with the `Dynamic` option." ?

**Comment by ConnorLand** [comment]
> Requires a technical review before merge.

I added a label to this which can be removed after the tech review passes.

**Review by Keraito — APPROVED** [review]

File: content/appstore/widgets/accordion.md (hunk 3 above)
Comment (IIsaku) [reply content/appstore/widgets/accordion.md]: It means that start as can also be defined with the help of an expression if you select the "Dynamic" option. A new expression property will appear to configure whether is should start as collapsed.

**Review by Adam-Dupaski — APPROVED** [review]

**Comment by Adam-Dupaski** [comment]
@ConnorLand please review my commits upon return, as a lot of restructuring was necessary for best practices and clarity.

**Pull request merged**